syncopy/nwanalysis/ST_compRoutines.py

```diff
@@ -191,9 +191,9 @@ class ST_CrossSpectra(ComputationalRoutine):
     def process_metadata(self, data, out):
 
         # Some index gymnastics to get trial begin/end "samples"
-        if data._selection is not None:
-            chanSec = data._selection.channel
-            trl = data._selection.trialdefinition
+        if data.selection is not None:
```

Comment:
I count on VScode did the heavy lifting here and you didn't have to do this manually like 100's of times :sweat_smile:

Reply:
Nope, luckily VS Code came to the rescue ;)